### PR TITLE
fix(libp2p): export perf protocol

### PR DESCRIPTION
**Pull request by mxinden**
## Description

As we do with all other protocols, make sure to expose `libp2p-perf` as well.

Related: https://github.com/libp2p/rust-libp2p/pull/3508#discussion_r1125546920.

## Change checklist

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] A changelog entry has been made in the appropriate crates

### CODE REVIEW

File: libp2p/CHANGELOG.md [hunk 1]
```diff
@@ -11,9 +11,13 @@
 - Introduce `libp2p::allow_block_list` module and deprecate `libp2p::Swarm::ban_peer_id`.
   See [PR 3590].
 
+- Introduce `libp2p::perf` module.
+  See [PR XXX].
+
 [PR 3386]: https://github.com/libp2p/rust-libp2p/pull/3386
 [PR 3580]: https://github.com/libp2p/rust-libp2p/pull/3580
 [PR 3590]: https://github.com/libp2p/rust-libp2p/pull/3590
+[PR XXX]: https://github.com/libp2p/rust-libp2p/pull/XXX
```
Comment (thomaseizinger) [inline libp2p/CHANGELOG.md:20]:
```suggestion
  See [PR 3693].

[PR 3386]: https://github.com/libp2p/rust-libp2p/pull/3386
[PR 3580]: https://github.com/libp2p/rust-libp2p/pull/3580
[PR 3590]: https://github.com/libp2p/rust-libp2p/pull/3590
[PR 3693]: https://github.com/libp2p/rust-libp2p/pull/3693
```

**Review by thomaseizinger — APPROVED** [review]
Makes sense.

**Comment by thomaseizinger** [comment]
Nit: All the examples in https://www.conventionalcommits.org/en/v1.0.0/#summary are using a lowercase commit message so I renamed your title.

**Comment by mxinden** [comment]
> Nit: All the examples in https://www.conventionalcommits.org/en/v1.0.0/#summary are using a lowercase commit message so I renamed your title.

:+1: still subconsciously using upper case. Please keep reminding me in case i forget.

File: libp2p/CHANGELOG.md (hunk 1 above)
Comment (thomaseizinger) [reply libp2p/CHANGELOG.md:20]: omg, I am an idiot and forgot to commit that :joy:

Comment (thomaseizinger) [reply libp2p/CHANGELOG.md:20]: Fixed in https://github.com/libp2p/rust-libp2p/pull/3694.

**Pull request merged**